readthedocs.yml

```diff
@@ -4,10 +4,11 @@ sphinx:
   configuration: docs/source/conf.py
 
 build:
-  image: latest
```

Comment:
Can you add the common top comment in this file as well, clarifying where the RTD project is located?

Copy paste from another jupyterhub project, such as jupyterhub/jupyterhub

Reply:
Sure! added duplicate comments to this file